src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+    </tr>
+    <tr>
+        <td><inlineCode>main</inlineCode></td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Indicates the primary JavaScript or HTML file, relative to the plugin"s installation directory. Supports HTML and JS files, such as <inlineCode>index.html</inlineCode> and <inlineCode>index.js</inlineCode>. If not specified (for deprecations), <inlineCode>main.js</inlineCode> is used.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>"main.js"</inlineCode></p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>icons</inlineCode></td>
+        <td><inlineCode>IconDefinition[]</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>An array of icons representing the overall plugin or panel icon. The host application uses these icons to present the plugin to the user. If the icons array is missing, a default icon will be used.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>[]</inlineCode> (no icons)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>host</inlineCode> *</td>
+        <td><inlineCode>HostDefinition</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The host object indicates the plugin"s compatibility with the host. Incompatible plugins will:</p>
+            <ul>
+                <li>fail to install if attempted in the given host</li>
+                <li>be invisible in the in-app plugin marketplace for the given host</li>
+                <li>be unavailable for update if the update is no longer compatible.</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>entrypoints</inlineCode> *</td>
+        <td><inlineCode>EntrypointDefinition[]</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The entrypoints array defines the entrypoints that the plugin provides.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>requiredPermissions</inlineCode></td>
+        <td><inlineCode>PermissionsDefinition</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Indicates the various permissions this plugin requires. These permissions are required before accessing certain API surfaces. Without these permissions in the manifest, API access may fail or throw an error. Some permissions require user consent.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no permissions)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>addon</inlineCode></td>
+        <td><inlineCode>object</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Addon definitions for hybrid plugins. See the hybrid plugin documentation for more information. A UXP Hybrid plugin is a UXP plugin that can access the power of C++ native libraries. This currently isn't supported in InDesign.</p>
```

Comment:
Replace this with
"Not supported in InDesign v18.5.
Addon definitions for hybrid plugins. A UXP Hybrid plugin is a UXP plugin that can access the power of C++ native libraries."